src/atn-editor/store/reducers/featureReducer.ts

```diff
@@ -0,0 +1,28 @@
+import { AnyAction } from 'redux';
+
+export interface StateType {
+  multiLanguage: boolean;
+  download: boolean;
+  [key: string]: boolean; // For dynamic feature access
+}
+
+//
+// Simply toggle the feature here
+//
+export const initialState: StateType = {
```

Comment:
Same here. Can we use proper names for all the state types?
```suggestion
export const initialState: FeatureReducerState = {
```